e2e/rbd.go

```diff
@@ -58,6 +58,8 @@ var (
 	appSmartClonePath      = rbdExamplePath + "pod-clone.yaml"
 	appBlockSmartClonePath = rbdExamplePath + "block-pod-clone.yaml"
 	snapshotPath           = rbdExamplePath + "snapshot.yaml"
+	rxAppPath              = rbdExamplePath + "fs-pod-rox.yaml"
```

Comment:
start with app..

Reply:
There is rawAppPath already! Hence I just used rxRawAppPath

Reply:
most of the variable names regarding the app starts with app.., so I would suggest to be consistent on the naming. If one other correction is required , please feel free to do so.